whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -55,21 +55,33 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.fail;
 
-@Tag("IntegrationTest")
+@Tag("ElasticSearchTest")
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class WhoisRdapServiceTestIntegration extends AbstractRdapIntegrationTest {
 
-    @Autowired
-    FullTextIndex fullTextIndex;
+    private static final String WHOIS_INDEX = "whois_rdap_service";
+
+    private static final String METADATA_INDEX = "metadata_rdap_service";
     @Autowired
     TestWhoisLog queryLog;
 
 
     @BeforeAll
     public static void beforeClass() {
+        System.setProperty("elastic.whois.index", WHOIS_INDEX);
+        System.setProperty("elastic.commit.index", METADATA_INDEX);
+        System.setProperty("fulltext.search.max.results", "10");
```

Comment:
There is WhoisRdapElasticServiceTestIntegration .. Can you just remove the  ES tests from here?

Reply:
Done